packages/spectacular/src/lib/feature-test-harness/feature-test-harness.ts

```diff
@@ -0,0 +1,74 @@
+import { NavigationExtras, UrlTree } from '@angular/router';
+
+/**
+ * A test harness for an Angular feature module. Includes an API to test
+ * as-a-user.
+ */
+export interface FeatureTestHarness {
+  /**
+   * Click the button with the specified label.
+   *
+   * @param label The label of the button.
+   */
+  clickButton(label: string): void;
```

Comment:
any issues if we leave it as click, we may have an anchor or even sometimes span with click action, nut sure if its valid scenario.

Reply:
That's a good point, @santoshyadavdev. Although this PR is about `bootstrapTestApplication`, not feature test harness which was just moved to a sub-directory as part of this PR. We will tweak feature test harness in a separate PR, so I'll resolve this comment, but thanks!